style.css

```diff
@@ -602,6 +1236,157 @@ body {
     width: 24px;
     height: 34px;
   }
+
+  .second-page {
+    min-height: 749px;
+  }
+
+  .second-page__img {
+    margin-right: 52px;
+  }
+
+  .second__img_purses {
+    width: 83%;
+    padding: 93px 0 0 114px;
+  }
+
+  .second__wrap {
+    margin: 40px 15px 0 0;
+  }
+
+  .second-page__article {
+    width: 310px;
+    padding: 32px 52px 35px 45px;
+    margin-bottom: 27px;
+  }
+
+  .second-page__title {
+    font-size: 35px;
+  }
+
+  .second-page__subtitle {
+    font-size: 14px;
+    letter-spacing: 0.6px;
+    margin-bottom: 48px;
+  }
+
+  .second-page__description {
+    font: 600 12px/17px 'Montserrat-Regular', sans-serif;
+    margin-bottom: 36px;
+    letter-spacing: .4px;
+  }
+
+
+  .second-page__color-options {
+    margin-bottom: 32px;
+  }
+
+  .eclipse {
+    width: 26px;
+    height: 26px;
+    margin-right: 12px;
+  }
+
+  .second-page__price {
+    margin-bottom: 42px;
+    font-size: 34px;
+  }
+
+  .second__quantity__input {
+    width: 11px;
+  }
+
+  .second__add-to-cart {
+    font-size: 11px;
+    padding: 2px 17px;
+    width: 211px;
+  }
+
+  .second__more-items {
+    font-size: 11px;
+    letter-spacing: -.1px;
+    padding-left: 4px;
+  }
+
+  .second__more-vector {
+    padding-left: 10px;
+    width: 45%;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 323px, #000 323px, #000 384px, #EBE3D8 384px);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)